content/docs/design/conditional-branch/conditional-branch.md

```diff
@@ -0,0 +1,178 @@
++++
+title = "Conditional Branch Requirement Overview and Design"
```

Comment:
How about calling this feature `Dynamic Conditional Concurrent`

Reply:
changed to Conditional Concurrency